### PR TITLE
Clean up behavior around edgectl connect and namespaces

### PR DESCRIPTION
**Pull request by ark3**
- Namespace, where teleproxy runs, is whatever the context says by default
- Manager namespace, where the manager runs, is ambassador by default
- Both can be set independently

### CODE REVIEW

File: cmd/edgectl/intercept.go [hunk 1]
```diff
@@ -262,8 +262,6 @@ func (cept *Intercept) removeMapping(p *supervisor.Process) error {
 	if cept.mappingExists {
 		p.Logf("%v: Deleting mapping in namespace %v", cept.ii.Name, cept.ii.Namespace)
 		delete := cept.cluster.GetKubectlCmd(p, "delete", "-n", cept.ii.Namespace, "mapping", fmt.Sprintf("%s-mapping", cept.ii.Name))
-		delete.Stdout = os.Stdout
-		delete.Stderr = os.Stderr
```
Comment (kflynn) [inline cmd/edgectl/intercept.go]: Hmmm. So what happens if this fails?

File: cmd/edgectl/intercept.go [hunk 2]
```diff
@@ -326,7 +324,6 @@ func MakeIntercept(p *supervisor.Process, out *Emitter, tm *TrafficManager, clus
 	}
 
 	manifest, err := json.MarshalIndent(&mapping, "", "  ")
-
```
Comment (kflynn) [inline cmd/edgectl/intercept.go]: Dude. Vertical whitespace is not a bad thing. :joy:

File: cmd/edgectl/notify.go [hunk 3]
```diff
@@ -7,10 +7,21 @@ import (
 	"github.com/datawire/ambassador/pkg/supervisor"
 )
 
-var notifyRAI *RunAsInfo
+var (
+	notifyRAI     *RunAsInfo
+	notifyEnabled = false
```
Comment (kflynn) [inline cmd/edgectl/notify.go]: Did I miss a way to turn this back on?

File: cmd/edgectl/intercept.go (hunk 1 above)
Comment (ark3) [reply cmd/edgectl/intercept.go]: The output gets logged anyhow (by `supervisor.Command`). This was causing it all to be doubled, half the lines with the logging headers and half without.

File: cmd/edgectl/intercept.go (hunk 2 above)
Comment (ark3) [reply cmd/edgectl/intercept.go]: I blame `gofmt`! I didn't remove those on purpose.

File: cmd/edgectl/notify.go (hunk 3 above)
Comment (ark3) [reply cmd/edgectl/notify.go]: Oh, good point. Not everything in my tree got committed; I should have left this out too. That said, notifications don't work on Linux (any longer), so I'm not too fussed about leaving it off until the next RC. Is that okay?

File: cmd/edgectl/intercept.go (hunk 1 above)
Comment (kflynn) [reply cmd/edgectl/intercept.go]: Ahhhh gotcha.

File: cmd/edgectl/intercept.go (hunk 2 above)
Comment (kflynn) [reply cmd/edgectl/intercept.go]: *!@&*!#& but whatever. :wink:

File: cmd/edgectl/notify.go (hunk 3 above)
Comment (kflynn) [reply cmd/edgectl/notify.go]: I'm OK with leaving it for the next RC. Let's talk about it later; I think the notifications could be very useful if tweaked a bit.

**Pull request merged**